package.json

```diff
@@ -45,12 +45,12 @@
     "@next/eslint-plugin-next": "^12.1.5",
     "@rushstack/eslint-patch": "^1.1.3",
     "@types/hosted-git-info": "^3.0.2",
-    "@types/lru-cache": "^7.6.1",
+    "@types/lru-cache": "^7.10.10",
     "@types/node": "^17.0.25",
     "@types/react": "^18.0.6",
     "@types/styled-components": "^5.1.25",
     "eslint": "^8.14.0",
-    "husky": "^7.0.4",
+    "husky": "^8.0.1",
```

Comment:
👋 @jurre! Seems like the diff includes changes from https://github.com/kachkaev/njt/pull/180. Can it be the case of Dependabot running in a dirty working directory? I did not modify Dependabot PRs myself.

Reply:
So I _think_ what happens is, we determine the sha to use when opening the PR, but then fetch whatever is in main when performing the update, and then open the PR against that (now outdated) sha. It seems like a pre-existing issue in Dependabot that we should definitely fix

Reply:
Ah right, makes sense. Thanks for investigating! 💯